python/ml_wrappers/model/openai_wrapper.py

```diff
@@ -11,14 +11,22 @@
     openai_installed = True
 except ImportError:
     openai_installed = False
+try:
+    from openai import AzureOpenAI, OpenAI
+except ImportError:
+    # Ignore the error, only used by new openai version
```

Comment:
Curious if we should have a user-facing warning statement instead of just silently continuing without importing?

Reply:
maybe at some later time we can start to print a warning - right now I think most people are probably still using older versions of this package anyway